text/3501-edition-2024.md

```diff
@@ -0,0 +1,88 @@
+[project group]: https://rust-lang.github.io/rfcs/2856-project-groups.html
+
+# Motivation
+[motivation]: #motivation
+
+So far, editions have been delivered on a three-year cadence, with the 2015, 2018, and 2021 editions.
+Many within the Rust project have been expecting this trend to continue.
+The three-year cadence seems to hit a reasonable balance of not too often, but still providing opportunities for potentially breaking changes to be introduced.
+
+By establishing a three-year cadence, this removes the ambiguity of what the expectations are around scheduling and releases of new editions.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## 2024 Edition
+
+The Rust Project intends to create a 2024 Edition.
+Currently, no specific changes are being announced in this RFC for this edition.
+Team members will be responsible for identifying changes they want to make and coordinate with the Edition project group.
+
+## Edition project group
+
+The Leadership Council will be responsible for forming a project group for coordinating each edition release.
+The Edition project group is responsible for:
+
+* Ensuring that the process follows the accepted policy, such as [RFC 3085] and this RFC.
+* Establishing a schedule for the edition ([example][example-schedule]).
+* Building consensus within the Edition project group on which changes will be accepted for the edition in coordination with the affected teams.
+* Coordinating with teams to ensure that edition changes are on track for release.
+* Making recommendations to the Leadership Council if the edition should be released in another year.
+* Making public announcements, such as blog posts to solicit support in creating the edition, calls for testing if needed, and informing users about the changes planned.
+* Ensuring that the requisite changes in tooling for changes are available in the nightly release, such as adding unstable edition flags.
+* Informing the relevant people to be aware of any issues related to the edition and tracking for getting those issues resolved.
+* Ensuring that sufficient testing, such as crater runs, gets done in time to be evaluated and for issues to be resolved.
+* Ensuring that automated migrations are in place, and should cover the majority of users.
+* Ensuring documentation, such as the [Edition Guide], is updated, ideally in time for users to begin testing before the release.
+* Ensuring that the stabilization process happens on schedule.
+* Monitoring adoption of the edition after the stabilization, and identifying issues that may need to be addressed after the release.
+* Continue to evaluate if the three year cadence makes sense, and potentially make recommendations to the Leadership Council for changes to the edition process.
+
+The Edition project group and the Leadership Council may approve minor changes to the process outlined above without public comment.
```

Comment:
I would personally prefer that we fully delegate this to the project group - I don't see a reason for diffusing responsibility. Can we do that?

Reply:
Sure!